packages/gasket-plugin-workbox/lib/utils.js

```diff
@@ -38,15 +38,15 @@ function getOutputDir(gasket) {
 }
 
 /**
- * Get the asset prefix from either next or workbox config.
+ * Get the asset prefix from next, workbox, or zone config.
  * If found in both, the workbox config will be used.
  *
  * @param {Gasket} gasket - Gasket
  * @returns {string} prefix
  */
 function getAssetPrefix(gasket) {
-  const { next = {}, workbox = {} } = gasket.config;
-  let { assetPrefix = '' } = next;
+  const { next = {}, workbox = {}, zone } = gasket.config;
+  let assetPrefix = zone || next.assetPrefix || '';
```

Comment:
We should probably remove `next.assetPrefix` from the equation here now that we can use `zone` for a common config setting:
```suggestion
  let assetPrefix = workbox.assetPrefix || zone || '';
```